src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating(){
```

Comment:
Having a default implementation of `isMutating()` is good as it reduced the amount of code; However choosing an arbitrary default value (`true` or `false`) is not recommended. Instead you can try declaring `isMutating()` as an `abstract` method with empty body which implies that sub-classes of `Command` would need to implement `isMutating()`.